api/external-api.adoc

```diff
@@ -3,18 +3,18 @@ Anfrage Gruppenurl::
 
 Anfrage:
 
-include::../build/generated-snippets/index/GroupController/get-group-url/path-parameters.adoc[]
+include::{snippets}/index/GroupController/get-group-url/path-parameters.adoc[]
```

Comment:
The advantage of the previous version is you can run it from command line.

Reply:
docToHtml task didn't really work